src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-OLTP-SYSBENCH.json

```diff
@@ -8,135 +8,169 @@
     },
     "Parameters": {
         "DatabaseName": "sbtest",
-        "DatabaseScenario": "Balanced",
-        "Duration": "00:05:00"
+        "NumTables": "10",
+        "DatabaseScenario": "Default",
+        "Duration": "00:05:00",
+        "SQLServerPackageName": "mysql-server",
+        "WorkloadGeneratorPackageName": "sysbench"
     },
     "Actions": [
         {
-            "Type": "SysbenchOLTPServerExecutor",
+            "Type": "SysbenchConfiguration",
```

Comment:
I would still expect the database initial creation, distribution across the disks and then the full population to be in the Dependencies. I don't think we need to support changing the database size on each iteration of the profile. If the user needs to support more than 1 database size on the same system, I recommend that we support this in the following way in the near term (vs. trying to do this mid-stream in the Actions):

How to test against different database sizes:
User will simply perform a VC reset (i.e. VirtualClient.exe clean, VirtualClient.exe reset) in between the runs. A reset/clean removes all packages that are not "built-in" packages, deletes the state files in the /state directory and deletes the log files in the /logs directory. The intention is to set VC back to the state it was in at the very beginning before being run.

We should support this behavior of course. So for example, if the user starts with a database that has 10 tables with 10,000,000 records each and runs. The user should then be able to do a VC clean and execute a second scenario where there are 40 tables with 40,000,000 records each. We should be able to handle the following in support of this:

1) Repeat installations of MySQL server software.
2) Repeat updates to the MySQL variables in the config file for the server.
3) Resetting the database. Given that Sysbench simply appends to any existing database tables, we either need to drop the DB and rebuild or have a way to check to see how many tables and records there are. The latter sounds complicated, so I vote for the former...keep it simple.